### PR TITLE
Update GDPR of the forms and add checkboxes

### PR DESCRIPTION
**Pull request by illusionOfParadise**
https://app.clubhouse.io/enmarche/story/4632/modifier-les-mentions-d-informations-sur-certains-formulaires-du-site

### CODE REVIEW

File: src/Form/EventRegistrationType.php [hunk 1]
```diff
@@ -34,6 +35,13 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             ->add('acceptTerms', CheckboxType::class, [
                 'mapped' => false,
             ])
+            ->add('personalDataCollection', CheckboxType::class, [
+                'required' => true,
```
Comment (ottaviano) [inline src/Form/EventRegistrationType.php]: It's default behavior

File: src/Form/EventRegistrationType.php [hunk 2]
```diff
@@ -34,6 +35,13 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             ->add('acceptTerms', CheckboxType::class, [
                 'mapped' => false,
             ])
+            ->add('personalDataCollection', CheckboxType::class, [
```
Comment (ottaviano) [inline src/Form/EventRegistrationType.php]: What about creat a new type: `AcceptPersonalDataCollectType` with this configuration? + add one template which will render the chebox element with label

File: templates/adherent/create_citizen_project.html.twig [hunk 3]
```diff
@@ -76,13 +76,19 @@
                         {{ form_widget(form._token) }}
                     {{ form_end(form) }}
                     <p class="text--gray text--summary text--small b__nudge--top">
-                        Les informations recueillies sur ce formulaire sont enregistrées dans un fichier informatisé par La République En Marche pour le référencement de Projets citoyens et la mise en relation entre Adhérents et Utilisateurs du site <a href="https://en-marche.fr">https://en-marche.fr/</a>.
+                        Les données recueillies sur ce formulaire sont traitées par LaREM afin de gérer les informations relatives aux projets citoyens et de permettre à LaREM de vous envoyer des communications politiques.
+                        </p>
```
Comment (ottaviano) [inline templates/adherent/create_citizen_project.html.twig]: CS: indent

File: src/Form/SummaryType.php [hunk 4]
```diff
@@ -111,6 +113,13 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                     ->add('availabilities', JobDurationChoiceType::class)
                     ->add('job_locations', JobLocationChoiceType::class)
                     ->add('professional_synopsis', TextareaType::class, ['filter_emojis' => true])
+                    ->add('personal_data_collection', CheckboxType::class, [
+                        'required' => true,
```
Comment (thomasbisignani) [inline src/Form/SummaryType.php]: The default value is already `true`, this line is not needed.

File: src/Form/SummaryType.php [hunk 5]
```diff
@@ -155,6 +164,13 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             case self::STEP_INTERESTS:
                 $builder
                     ->add('member_interests', MemberInterestsChoiceType::class)
+                    ->add('personal_data_collection', CheckboxType::class, [
+                        'required' => true,
```
Comment (thomasbisignani) [inline src/Form/SummaryType.php]: Same here

File: src/Form/SummaryType.php [hunk 6]
```diff
@@ -177,6 +193,13 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                     ->add('viadeo_url', UrlType::class, [
                         'required' => false,
                     ])
+                    ->add('personal_data_collection', CheckboxType::class, [
+                        'required' => true,
```
Comment (thomasbisignani) [inline src/Form/SummaryType.php]: Same here

File: templates/newsletter/subscription.html.twig [hunk 7]
```diff
@@ -37,6 +37,17 @@
                         {{ form_widget(form.postalCode, { 'attr': { 'class': 'form--full' } }) }}
                     </div>
 
+                    <div class="b__nudge--top text--small form__checkbox form__checkbox--large">
+                        {{ form_row(form.personalDataCollection, { label: 'J’ai lu et j’accepte les mentions d’information relatives au recueil de mes données personnelles ci-dessous :', label_attr: { id: 'field-personal-data-collection' } }) }}
```
Comment (thomasbisignani) [inline templates/newsletter/subscription.html.twig]: Can you use the correct `'` char instead of `’` in all templates ?

**Pull request merged**